src/systems/ec/implementations/react/templates/pages/research-area-hub/src/ResearchAreaHubPage.jsx

```diff
@@ -0,0 +1,1301 @@
+                      <span className="ecl-u-type-s ecl-u-ml-s">
+                        Brussels, Belgium
+                      </span>
+                    </UnorderedListItem>
+                  </UnorderedList>
+                </div>
+              </article>
+
+              <article className="ecl-u-border-bottom ecl-u-border-color-grey-15 ecl-u-d-flex ecl-u-pv-m">
+                <div className="ecl-u-flex-grow-0 ecl-u-mr-m">
+                  <DateBlock
+                    variant="past"
+                    dateTime="2019-11-13"
+                    day="13"
+                    month="Nov"
+                    monthFull="November"
+                    year="2019"
+                  />
+                </div>
+                <div className="ecl-u-flex-grow-1">
+                  <div className="ecl-u-d-inline-block ecl-u-type-s ecl-u-bg-grey-25 ecl-u-pa-2xs">
+                    Past
+                  </div>
+                  <div className="ecl-u-type-prolonged-m ecl-u-type-bold ecl-u-mt-xs">
+                    <Link
+                      href="/example"
+                      variant="standalone"
+                      label="2019 Global Ocean Science Education Workshop"
+                    />
+                  </div>
+                  <UnorderedList
+                    variant="no-bullet"
+                    className="ecl-u-mt-s ecl-u-type-color-grey-75"
+                  >
+                    <UnorderedListItem className="ecl-u-d-flex ecl-u-align-items-center">
+                      <Icon shape="general--location" size="m" />
+                      <span className="ecl-u-type-s ecl-u-ml-s">
+                        Washington, United States
+                      </span>
+                    </UnorderedListItem>
+                    <UnorderedListItem className="ecl-u-d-flex ecl-u-align-items-center">
+                      <Icon shape="ui--external" size="m" />
+                      <span className="ecl-u-type-s ecl-u-ml-s">
+                        External event
+                      </span>
+                    </UnorderedListItem>
+                  </UnorderedList>
+                </div>
+              </article>
+              <Link
+                className="ecl-u-mt-l ecl-u-type-bold"
+                variant="standalone"
+                label="More events"
+                href="/example"
+                icon={{
+                  shape: 'ui--corner-arrow',
+                  size: 'xs',
+                  transform: 'rotate-90',
+                }}
+              />
+
+              <h2
+                className="ecl-u-type-heading-2 ecl-u-mb-l ecl-u-mt-lg-3xl ecl-u-type-color-black"
+                id="contact"
+              >
+                Contact
+              </h2>
+
+              <div className="ecl-row ecl-u-mt-xs">
+                <div className="ecl-col-12 ecl-col-md-6">
+                  <article className="ecl-u-pv-m ecl-u-height-100 ecl-u-box-sizing-border">
```

Comment:
gap between 'Contact' parent heading is too high, 
therefore I think we need to remove top margin here and use `ecl-u-pb-m` 
and also remove top margin for the next `<h2>` in column.

Reply:
looking more closely, this should be a `<h3>`

Reply:
hopefully better now